src/Resources/skeleton/command/Command.tpl.php

```diff
@@ -22,19 +26,22 @@ protected function configure()
         ;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $io = new SymfonyStyle($input, $output);
+        $symfonyStyle = new SymfonyStyle($input, $output);
```

Comment:
I'm not sure I like this variable name. First, we use `$io` everywhere else in Symfony. Second, this object is not really a "Symfony style" but an "input + output object that displays things using the Symfony style".